docs/source/usage/cijoe-script.py

```diff
@@ -0,0 +1,7 @@
+from cijoe.cli.cli import cli_interface
+
+def main(args, cijoe, step):
+    cijoe.run("echo 'Hello World!'")
```

Comment:
Can you expand this with:

```python
err, _ = cijoe.run("echo 'Hello World!'")
return err
```

I don't have a great example right now, but it would be really nice for the this "skeleton" script to show what one might commonly do, so perhaps even something like:

```python
err, state = cijoe.run("echo 'Hello World!'")

# Do this to demonstrate how to process command-output
if "Hello" not in state.output():
   ... show some logging here ...
   
return err
```

What do you think?